targets/PythonSdk/Archive/PlayFabBaseClasses.py

```diff
@@ -1,14 +1,18 @@
 class PlayFabBaseObject():
     pass
 
+"""
 # This is a base-class for all Api-request objects.
```

Comment:
># [](start = 0, length = 1)

As Hong mentioned, it looks like we don't need the single-line comment characters anymore since we replace them with multiline.

Reply:
/agree